### DISCUSSION TITLE
I get the following issue when launching docker，how to

### DISCUSSION THREAD

**Discussion by gwfs521** [ramonvc/freegpt-webui#190] [Q&A]
run -p 1338:1338 ramonvc/freegpt-webui
docker: Error response from daemon: Ports are not available: exposing port TCP 0.0.0.0:1338 -> 0.0.0.0:0: listen tcp 0.0.0.0:1338: bind: An attempt was made to access a socket in a way forbidden by its access permissions.
how to resolve it？thanks

**Comment by nqf** [comment]
https://github.com/fantasy-peak/cpp-freegpt-webui
